test/test_cuda.py

```diff
@@ -1733,6 +1733,46 @@ def test_streaming_backwards_device_transfer(self):
         self.assertTrue(a.grad.sum().item() == 4 * size)
         self.assertTrue(b.grad.sum().item() == 4 * size)
 
+    def test_streaming_backward_sync_graph_root(self):
```

Comment:
How long is this test? Should be we consider a `@slowTest` here?

Reply:
It's quick, 2 seconds when i run in isolation and profiling shows the vast majority is cuda context initialization.  No need for @slowTest imo.